app/src/main/java/edu/uc/hickmadc/customnotifications/adapters/GroupsAdapter.kt

```diff
@@ -49,10 +50,15 @@ class GroupsAdapter(private val dataSet: ArrayList<Group>) :
         }
     }
 
+    /**
+     * Handles the comparison between groups.
+     * @return [areItemsTheSame]
+     * @return [areContentsTheSame]
```

Comment:
Added return annotations, interesting technical suggestion here, you can actually [wrap things like this] to reference certain bits of code and the javadoc goes with it!

Reply:
There should only be one return annotation.  And, if adding it, make it descriptive.  Otherwise, it's not adding value.